front/src/components/AdminCardVoter/AdminCardVoter.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+import { Heading } from '@chakra-ui/react';
+import { Box } from '@chakra-ui/react';
+import { Text } from '@chakra-ui/react';
+
+export default function AdminCardVoter( { num, address }:any) {
+  return (
+	<>
+		<Box>
+			<Heading size='xs' textTransform='uppercase'>
+			Voter #{ num }
+			</Heading>
+			<Text pt='2' fontSize='sm'>
+			⟠ { address }
+			</Text>
+		</Box>
+	</>
+  );
```

Comment:
```suggestion
		<Box>
			<Heading size='xs' textTransform='uppercase'>
			Voter #{ num }
			</Heading>
			<Text pt='2' fontSize='sm'>
			⟠ { address }
			</Text>
		</Box>
  );
```
Pas besoin du Container, tu as Box déjà